tests/unit/web/request.test.ts

```
@@ -110,6 +110,59 @@ describe('Request Utils', () => {
     });
   });
 
+  describe('parseRequestBody - additional edge cases', () => {
+    function createMockRequestWithBuffer(chunks: Buffer[]): IncomingMessage {
+      const stream = new Readable({
+        read() {
+          chunks.forEach(chunk => this.push(chunk));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)